modules/prepare-operator-upgrades.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-operator.adoc
+:_module-type: CONCEPT
+[id="prepare-operator-upgrades_{context}"]
+= Preparing to upgrade
+
+[role="_abstract"]
+Before you upgrade {rh-rhacscs-first} version, you must:
+
+* verify that you are running the latest version 3.74 of the {product-title-short} Operator.
```

Comment:
"latest version 3.74" ? What about "latest release of the 3.74 version of the RHACS Operator"?